src/main/java/uk/ac/ebi/ena/webin/cli/context/SequenceToolsXmlWriter.java

```diff
@@ -127,6 +128,10 @@ public Map<SubmissionBundle.SubmissionXMLFileType, String> createXml(
       analysisAttributesE.addContent(submissionToolVersionAnalysisAttributeE);
     }
 
+    if (manifest instanceof PolySampleManifest) {
+      addPolySampleAttributes((PolySampleManifest) manifest, analysisAttributesE);
+    }
```

Comment:
Can you help me understand the need to change this class when you have created a dedicated XML writer class for polysamples?